sklearn/gaussian_process/gpr.py

```diff
@@ -150,7 +148,12 @@ def fit(self, X, y):
 
         y : array-like, shape = (n_samples, [n_output_dims])
             Target values
-
+            
+        sample_alpha : float or array like, shape = (n_samples,), optional
```

Comment:
Should we allow a float here? What happens if the user specified a float in ``__init__`` and an array here?